### PR TITLE
fixed sass import paths

### PR DESCRIPTION
**Pull request by delta98**
updated badge position now the import works and changes it for tabs to relative
added .sass-cache to gitignore

### CODE REVIEW

**Comment by dciccale** [comment]
this fixes the problem! 

But only if i change in the gulpfile `sass` task to compile `gulp.src(['./scss/ionic-material-design-lite.scss'])` instead of `gulp.src(paths.sass)` so this has to be changed in line 37 but in the new `gulpfile.js` and this branch has the old one

**Comment by dciccale** [comment]
if you merge this PR i and fix the rest of it by updating the gulpfile

**Comment by dciccale** [comment]
or you can do it yourself rebasing this branch, whatever :)

**Comment by delta98** [comment]
Glad that's fixed the issue, I will rebase and merge into master.

Thanks for your help

**Comment by delta98** [comment]
@dciccale checkout master and make sure it's working for you, I've had to update the sass function as it wasn't working with `gulp-load-plugin` correctly.

File: gulpfile.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ gulp.task('minor-release', ['bump']);
 gulp.task('major-release', ['major-bump']);
 
 gulp.task('scripts', function() {
-    return gulp.src(paths.js)
+    return gulp.src('./scss/ionic-material-design-lite.scss')
```
Comment (dciccale) [inline gulpfile.js]: ups, careful this is the scripts task sory i think i messed you up. this has to be `paths.js` and the sass task `gulp.src(['./scss/ionic-material-design-lite.scss'])`

**Comment by dciccale** [comment]
with that last change i think everything will be good to go!

**Comment by dciccale** [comment]
PR https://github.com/delta98/ionic-material-design-lite/pull/40

File: gulpfile.js (hunk 1 above)
Comment (delta98) [inline gulpfile.js]: gotcha! fixed.

**Pull request merged**